packages/big-design/src/components/Pagination/Pagination.tsx

```diff
@@ -90,11 +96,10 @@ export const Pagination: React.FC<PaginationProps> = memo(
       return onItemsPerPageChange(Number(item.hash));
     };
 
-    const showRanges = () => {
-      return itemRange.start === itemRange.end
-        ? `${itemRange.start} of ${totalItems}`
-        : `${itemRange.start} - ${itemRange.end} of ${totalItems}`;
-    };
+    if (getRangeLabel === null) {
+      getRangeLabel = (start: number, end: number, totalItems: number): string =>
+        start === end ? `${start} of ${totalItems}` : `${start} - ${end} of ${totalItems}`;
+    }
```

Comment:
How about we extract this outside of the component (same file) and default prop the `getRangeLabel`? I would also love to get rid of the ternary statement too 😬 
```tsx
const DEFAULT_GET_RANGE_LABEL: PaginationProps['getRangeLabel']  = (start, end, totalItems) => {
  if (start === end) {
    return `${start} of ${totalItems}`;
  }

  return `${start} - ${end} of ${totalItems};
}
```